regolith/helpers/u_milestonehelper.py

```diff
@@ -8,7 +8,7 @@
 from regolith.helpers.basehelper import DbHelperBase
 from regolith.fsclient import _id_key
 from regolith.tools import all_docs_from_collection
-
+from regolith.dates import get_due_date
```

Comment:
is there a ``get_due_date`` in regolith.dates?  do we even want one?  It seems too specific.

Why don't we use ``get_dates`` which is a nice function with lots of tests.  It looks for ``begin_date``, ``end_date``, and ``date``, so all we have to do to use it here is to add an item temporarily in the milestone documents that is date:
```
milestone['date']=milestone.get("due_date")
```
sthg like that?

Reply:
yes, there is a `get_due_dates` in regolith.dates, and it seems to have tests as well!
it does seems pretty specific, and I could change it to `get_date` if you prefer. 
Some milestones have a date field though, so maybe it would be better to temporarily use begin_date ?

Reply:
Oh, I see.  Go ahead and use it then!

Please also do a service to the project and pick a function or two in dates.py or tools.py and write tests for it.  We need to get the coverage up.

Reply:
sure! do I do this tests and add to this PR or should I do it separately?